### ISSUE TITLE
Addons not being launched

### ISSUE THREAD

**Issue by leopupin** [Healix/Gw2Launcher#58]
After the first run, wich is a "normal mode" run, the addons aren't loaded when I log into the game, no matter what I do, even if I manually insert them into their folder after using the virtual mode for managing the folders. (even when selecting Basic mode, it will not work either)

**Comment by Healix** [comment]
If you want to use addons and run multiple accounts, you need to enable localized execution under Settings > Guild Wars 2 > Management.

**Comment by TheVakman** [comment]
FYI @Healix, to piggyback off this issue as it sounds similar - if I enable localized execution, I can't load multiple copies of the game at once with add-ons.

When I am watching the Gw2-64.exe in Task Manager when I try to load multiple instances, the memory usage leaks on the latest one launched and climbs until it crashes. If I turn off the localized execution, they launch fine with multiple copies running but no add-ons. If I enable localized execution and re-sync, I can load the games individually with add-ons working but if I try to launch another instance, it does what I mentioned above. 

It worked for me one time but I'm not sure if it was a fluke or what combination of settings I had to get it working. Basic/Virtual doesn't seem to make a difference (but if I choose "Virtual", then I need to manually copy the symlink for GShade. I have d912pxy, ArcDPS and GShade installed if you wanted to replicate.

**Comment by Healix** [comment]
d912pxy causes that - it requires exclusive access to its folder, so once one account is using it, other launches will stall. To use it on multiple accounts, set "localized execution" to "full".

**Comment by Sunriz3D** [comment]
Is it normal for GW2Launcher to copy Gw2.dat aswell when enabling localized execution ? It creates a folder of 60gb for each account.

**Comment by Healix** [comment]
Yes, when it's set to full, it will replicate the entire folder. These aren't real files, it's just giving multiple paths to the same set of files - notice it has no effect on your actual disk usage.

**Comment by Sunriz3D** [comment]
> Yes, when it's set to full, it will replicate the entire folder. These aren't real files, it's just giving multiple paths to the same set of files - notice it has no effect on your actual disk usage.

Ah i see, like symlinks. Thanks for the answer i didn't watch the total disk usage tbh. Love your tool!
